db.py

```diff
@@ -0,0 +1,51 @@
+import psycopg2
+from psycopg2 import Error
+
+
+def db_connection():
+        try:
+                connection = psycopg2.connect(user="user_1",
+                                              password="test123",
+                                              host="127.0.0.1",
+                                              port="5432",
+                                              database="storemanager_db")
+                connection.autocommit = True
+                return connection
+        except (Exception, psycopg2.DatabaseError) as error:
+                print("Error while connecting to PostgreSQL", error)
+
+
+def create_tables():
+        cursor = db_connection().cursor()
+      
+        salestable = """ 
```

Comment:
use underscore to separate your variable names. eg sales_table

Reply:
Hello Boniface,

Thank you for this worthy observation and for taking the time to look through my work.

Regards
Steven Ennis